php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -115,7 +115,8 @@ class NDB_BVL_InstrumentStatus
      */
     function getDataEntryStatus()
     {
-        return $this->_status['Data_entry'];
+        if (isset($this->_status['Data_entry'])) return $this->_status['Data_entry'];
```

Comment:
Isn't it easier to ensure there is a Data_entry var? (presumably it's missing for Json instruments which should have it)